fedimintd/src/encrypt.rs

```diff
@@ -66,7 +67,11 @@ pub fn get_key(password: Option<String>, salt_path: PathBuf) -> LessSafeKey {
     let algo = pbkdf2::PBKDF2_HMAC_SHA256;
     pbkdf2::derive(
         algo,
-        ITERATIONS.unwrap(),
+        if std::env::var("FM_TEST_FAST_WEAK_CRYPTO").as_deref() == Ok("1") {
```

Comment:
We should probably begin to document these "dev flags" and eventually deactivate them for production builds.